### ISSUE TITLE
intel opencl gpu drivers still broken # missing security check to tell the user -> dont use this

### ISSUE THREAD

**Issue by realSnoopy** [hashcat/hashcat#3477]
please also see this thread
https://hashcat.net/forum/thread-11055-post-56450.html

hashcat (v6.2.6)

intel opencl gpu drivers seems to be still broken, but it seems the security check for this type of device/driver setup is missing ( i believe there was such a check in hashcat 6.2.5) 

in the thread you can see that running an attack will result in non cracking using a simple mask-attack, output hashcat -I taken from the user (thread)

OpenCL Info:
============

OpenCL Platform ID #1
  Vendor..: Intel(R) Corporation
  Name....: Intel(R) OpenCL HD Graphics
  Version.: OpenCL 3.0

  Backend Device ID #1
    Type...........: GPU
    Vendor.ID......: 8
    Vendor.........: Intel(R) Corporation
    Name...........: Intel(R) UHD Graphics
    Version........: OpenCL 3.0 NEO

**Comment by jsteube** [comment]
Indeed, the main problem with the Intel GPU drivers is that they sometimes work and with the next version they don't work anymore for no reason. It's purely random. This goes on and on since many years.

We have to choose between false negatives or blocking the user at all from trying. In the past it was easy because the intel JiT compiler segfaulted during kernel compile, but at some point that fixed and we reanabled them. 

So it's not a technical issue anymore. One could say a false negative is the worst thing that could happen, but then if hashcat just refuse to use Intel GPU, the user would just switch to a different tool that allows it.
